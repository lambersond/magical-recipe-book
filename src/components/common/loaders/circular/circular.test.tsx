import { render } from '@test-utils'
import { CircularLoader } from './circular'

describe('components/common/loaders/circular', () => {
  it('should render', () => {
    expect(render(<CircularLoader />).asFragment()).toMatchSnapshot()
  })
})
