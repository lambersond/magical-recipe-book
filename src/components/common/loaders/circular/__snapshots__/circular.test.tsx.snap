// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/common/loaders/circular should render 1`] = `
<DocumentFragment>
  <div
    class="text-primary inline-block h-8 w-8 animate-spin rounded-full border-4 border-solid border-current border-e-transparent align-[-0.125em] text-surface motion-reduce:animate-[spin_1.5s_linear_infinite]"
  >
    <span
      class="!absolute !-m-px !h-px !w-px !overflow-hidden !whitespace-nowrap !border-0 !p-0 ![clip:rect(0,0,0,0)]"
    >
      Loading...
    </span>
  </div>
</DocumentFragment>
`;
